packages/blade/src/components/Spinner/_decisions/decisions.md

```diff
@@ -0,0 +1,62 @@
+# Spinner <!-- omit in toc -->
+A spinner is an element with a looping animation that indicates loading is in process.
+
+<img src="./spinner-thumbnail.png" width="150" />
+
+## Design
+- [Figma - Spinner](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=14825%3A203592)
+
+
+## BaseSpinner
+A BaseSpinner will contain some extra props like `intent`. This will not be exposed to the consumer and will only be used for internal components.
+
+### API
+
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| accessibilityLabel | `string` | `undefined` | The accessibility label (aria-label) for the spinner. | ✅ |
+| intent | `positive`,`negative`, `notice`, `information`, `neutral` | `undefined` | The intent of the spinner to be rendered. |  |
+| contrast | `low`, `high` | `low` | The contrast of the spinner to be rendered. |  |
+| size | `small`, `medium`, `large` | `medium` | The size of the spinner to be rendered. |  |
+
+
+## Spinner
+A wrapper around Spinner which will be exposed to the consumer to be used as a Spinner. This component will not allow passing `intent` to the Spinner.
+
+### API
+
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| accessibilityLabel | `string` | `"Loading"` | The accessibility label (aria-label) for the spinner. |
```

Comment:
This is more of a question for design but have we considering adding an option for a label below the Spinner? There are many use cases of spinners being accompanied by labels like "Loading", "Searching" etc. and instead of a developer adding an extra div and then centrally aligning it to match the positioning of the spinner (which will be a repeated effort), maybe the design system could expose a prop for that? @saurav12 

This is also helpful for accessible users who can't perceive motion and rely on text content to grasp meaning.

Reply:
There was text before but they've removed it. @saurav12 can add more